.github/workflows/makefile-lint.yml

```
@@ -23,15 +38,25 @@
+            local line_end="$3"
+            printf "::error file=%s,line=%s,endLine=%s,title=VALIDATION_ERROR::ERROR %s\n" \
+              "${file_stub}" "${line_start}" "${line_end}" "${msg}"
+          }
+          go version || { print_error "Go installation failed." 40 41 ; exit 126; }
+          yamllint --version || { print_error "Yamllint installation failed." 40 41 ; exit 126; }
+          pandoc --version || { print_error "Pandoc installation failed." 40 41 ; exit 126; }
       - name: Lint Workflow YAML
         if: ${{ success() }}
+        env:
+          YAML_ARGS: "${{ vars.YAML_ARGS }}"
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:10: Double quote to prevent globbing and word splitting

(shellcheck)